test/Feed.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Feed from '../client/src/Feed';
+import Review from '../client/src/Review';
+
+describe('Feed Module Tests', () => {
+   it('renders 5 <Review /> components', () => {
+      const wrapper = shallow(<Feed />);
+      expect(wrapper).toContainMatchingElements(5, 'Review');
```

Comment:
Ah this is a smart way of doing this! I was really struggling figuring out how to work with the tests when I needed to reference multiple components